typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -120,32 +135,92 @@ export abstract class HyperlaneAppGovernor<
       SubmissionType.SIGNER,
       new SignerMultiSend(this.checker.multiProvider, chain),
     );
-    const owner = await resolveAccountOwner(
-      this.checker.multiProvider,
-      chain,
-      this.checker.configMap[chain].owner,
-    );
+    let safeOwner: Address;
+    if (typeof this.checker.configMap[chain].owner === 'string') {
+      safeOwner = this.checker.configMap[chain].owner as Address;
+    } else {
+      safeOwner = (this.checker.configMap[chain].owner as AccountConfig).owner;
```

Comment:
isnt this an address on a different chain?

Reply:
yes, that's correct. But in the inferCallSubmissionType for ica submission, I use origin chain as a param.